python/lsst/cp/pipe/utilsEfd.py

```diff
@@ -202,11 +206,67 @@ def selectTimeSeries(self, topicName, fields=[], startDate=None, endDate=None):
         table = Table(rows=series["values"], names=series["columns"], dtype=tableDtype)
         table["time"] = Time(table["time"], scale="utc")
         table.sort("time")
```

Comment:
Can you add a comment here about what the `time` column is and what the `private_sndStamp` column is and why we would like one over the other?